routes/users.rb

```diff
@@ -76,19 +81,29 @@
     session_token = result["token"]
     netid = result["user"]["name"]
 
-    user = User.first(netid: netid)
-
-    unless user
-        # Fetch user info, fill in what we can, and then return with token
-        # Should be mostly for development, where we don't have real user data
-        user_data = Auth.get_user_info(session_token)
-
-        user = User.create(
-            netid: netid,
-            first_name: user_data["user"]["first-name"],
-            last_name: user_data["user"]["last-name"],
-            is_member: true # must be an ACM user to have gotten here
-        )
+    if result["user"].to_json == local_fake_user.to_json
```

Comment:
This seems really weird to have fake user stubs in production code, is there not a way to have a stub service and add it to the dev manifest? Also what happens if someone uses these creds, they are public